### ISSUE TITLE
No Ubi-Canvas toolbar

### ISSUE THREAD

**Issue by WatWatWater** [byvar/ubi-canvas#15]
When I open up the scene, it looks like in the screenshot, but there is no Ubi-Canvas toolbar. I have opened up the scene, but I'm not sure if it is in the right version, as I could not install Unity 2020.1

**Comment by RayCarrot** [comment]
It seems the readme was a bit outdated before - I fixed it now. The recommended Unity version is 2021.3.6. Most likely the menu in the toolbar not showing is an issue with you using a different version than this project targets. Could you try installing 2021.3.6 and see if that fixes it?

**Comment by WatWatWater** [comment]
Yep, that worked, but I think I don't have the game directories, or don't know where they are.

**Comment by RayCarrot** [comment]
You need to extract the .ipk files in the game installation and then set that as the game directory.
I should also note Origins isn't fully supported yet. Legends is though and the demo should work.

**Comment by WatWatWater** [comment]
Well, the files are there.. But how do I extract them, and to where?

**Comment by RayCarrot** [comment]
You can use the Rayman Control Panel to extract them. There are also quickbms scripts out there to do it. The files can then be placed anywhere. You just need to specify its location in ubi-canvas.

**Comment by CattyCrashy** [comment]
I am also having the same issue Too, where I try some other versions of Unity within Ubi (Including 2021.3.6, 2022.3.7 and 2022 3.8), but for no no reasons, It still not working for me, the Toolbar missing issue.

**Comment by RayCarrot** [comment]
The latest version of ubi-canvas only supports 2022.3.7. If you tried opening it in another version first it might have broken some things. I would try to re-download ubi-canvas and only open with 2022.3.7.

**Comment by CattyCrashy** [comment]
Oky gotcha.
